src/span.js

```diff
@@ -222,14 +296,11 @@ export default class Span {
     }
 
     /**
-     * Indicates that the unit of work represented by the span is complete or
-     * has otherwise been terminated.
-     *
-     * All Span objects must have finish() called on them before they are
-     * reported to the backend implementation.
-     *
-     * Once `finish()` is called on a Span object, the behavior of all methods
-     * on the object is considered undefined.
+         * Sets the end timestamp and finalizes Span state.
+         *
+         * With the exception of calls to Span.context() (which are always allowed),
+         * finish() must be the last call made to any span instance, and to do
+         * otherwise leads to undefined behavior.
```

Comment:
Tabs?

(By the way, I should add an issue to add `eslint` to this project. It has a `--fix` option that takes care of a lot of things like this automatically)